src/services/playbook.ts

```diff
@@ -0,0 +1,35 @@
+import joi from 'joi'
+import { Logger } from './console'
+import { Command } from 'commander'
+import { formatJoiError } from './utils'
+
+const playbookSchema = joi.object({
+    name: joi.string().required(),
+    on: joi.array().items(
+        joi.object({
+            daemonId: joi.string(),
+            levels: joi.array().items(joi.string()).min(1),
+        })
+    ),
+    tasks: joi.object().required(),
+})
+
+export const ValidateAndReadPlaybook = (
```

Comment:
Please name the function accordingly of what is doing. 

this function seems to be checking if playbook is valid or not. so 

`isValidPlaybookManifest` looks as a better name

Also by convention functions in JS start with lowercase :)